cumulusci/tasks/salesforce/metadata_etl/base.py

```diff
@@ -11,10 +13,17 @@
 from cumulusci.core.config import TaskConfig
 
 
+class MetadataOperation(enum.Enum):
+    DEPLOY = "deploy"
+    RETRIEVE = "retrieve"
+
+
 class BaseMetadataETLTask(BaseSalesforceApiTask):
     deploy = False
     retrieve = False
 
+    __metaclass__ = abc.ABCMeta
```

Comment:
@davidmreed This is the Python 2 way of specifying metaclasses and I don't think it has special meaning in Python 3. The Python 3 way puts them alongside the base classes: https://docs.python.org/3/reference/datamodel.html#metaclasses

Reply:
Fixed in https://github.com/SFDO-Tooling/CumulusCI/pull/1544